src/stories/commits/CommitList.stories.mdx

```diff
@@ -9,7 +9,7 @@ import { Meta, Story } from '@storybook/addon-docs/blocks';
 ## Overview
 
 The **CommitList** component is a data table that shows the GitHub commit
-history of a **task**. This makes it possible to see at a glance how a **task**
+history of a **Task**. This makes it possible to see at a glance how a **Task**
```

Comment:
I'm not sure what we prefer here. I've seen `` `task` ``, `**task**`, and `**Task**`. I don't know that it matters, but might be worth standardizing.

Reply:
Good idea.  I've used `task` to refer to props used by a component, and **Task** for objects in the app, but not sure if there needs to be a distinction between them.

Reply:
yeah, I agree that these should be consistent. I've been thinking this needs some editorial guidelines. Maybe that is its own story?

Reply:
@dvdherron we can maybe add it as a checklist item on [I want a component index](https://trello.com/c/IeGjjRAH)?

Reply:
done